src/mapml/features/feature.js

```diff
@@ -240,6 +240,11 @@ export var Feature = L.Path.extend({
     this._outline = [];
     for (let coords of this._markup.querySelectorAll('map-coordinates')) {
       let nodes = coords.childNodes, cur = 0, tempDiv = document.createElement('div'), nodeLength = nodes.length;
+      for(let i = 0; i < nodes.length; i++){
+        if(nodes[i].textContent.trim().length === 0){
+          nodes[i].remove();
+        }
+      }
```

Comment:
Do you need to loop through all to childNodes or could you get away with just checking just the first and last?

Reply:
It seems that I only need to check the last one